src/clue.js

```diff
@@ -2,29 +2,71 @@
+const drOrchid={firstName:"doctor",lastName:"Orchid",occupation:"Scientist",age:26,description:"PhD in plant toxicology. Adopted daughter of Mr. Boddy", image:"http://www.radiotimes.com/uploads/images/Original/111967.jpg",color:"white"}
+const profPlum={firstName:"Victor", lastName:"Plum",occupation:"Designer",age:22,description:"Billionaire video game designer",image:"https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg", color:"purple"}
+const missScarlet={firstName:"Kassandra",lastName:"Scarlet", occupation:"Actor",age:31,description:"She is an A-list movie star with a dark past", image:"https://www.radiotimes.com/uploads/images/Original/111967.jpg",color:"red"}
+const mrsPeacock={firstName:"Eleanor",lastName:"Peacock",occupation:"Socialite", age:36,description:"She is from a wealthy family and uses her status and money to earn popularity",image:"https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",color:"blue"}
+const mrMustard={firstName:"Jack",lastName:"Mustard",occupation:"Retired Football player",age:62,description:"He is a former football player who tries to get by on his former glory",image:" https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg", color: "yellow"}
+
+suspectsArray = [mrGreen,drOrchid,profPlum,missScarlet,mrsPeacock,mrMustard]
 
 // Rooms Array
 
-const roomsArray = [];
+const roomsArray = [{name:"DiningRom"},
+                    {name:"Conservatory"},
+                    {name:"Kitchen"},
+                    {name:"Study"},
+                    {name:"Library"},
+                    {name:"BilliardRoom"},
+                    {name:"Lounge"},
+                    {name:"Ballroom"},
+                    {name:"Hall"},
+                    {name:"Spa"},
+                    {name:"LivingRoom"},
+                    {name:"Observatory"},
+                    {name:"Theater"},
+                    {name:"GuestHouse"},
+                    {name:"Patio",}];
 
 // Weapons Array
 
-const weaponsArray = [];
+const weaponsArray = [{name:"rope", weight:10},
+                      {name:"knife", weight:8},
+                      {name:"candlestick", weight:2},
+                      {name:"dumbbell", weight:30},
+                      {name:"poison", weight:2},
+                      {name:"axe", weight:15},
+                      {name:"bat", weight:13},
+                      {name:"trophy", weight:25},
+                      {name:"pistol", weight:20}
+];
 
 
 // ITERATION 2
 
-function selectRandom() {}
-
-function pickMystery() {}
+function selectRandom(array) {
 
+  return array[Math.floor(Math.random() * array.length)]
+    
+  }
+  
+ 
+function pickMystery() {
 
+let mistery={
+ suspect:selectRandom(suspectsArray),
+ weapon:selectRandom(weaponsArray),
+ rooms:selectRandom(roomsArray),
+}
+return mistery
+}
 // ITERATION 3
 
-function revealMystery() {}
-
+function revealMystery() {
 
+}
```

Comment:
function revealMystery(mys) {
  return `${mys.suspect.firstName} ${mys.suspect.lastName} killed Mr. Boddy using the ${mys.weapon.name} in the ${mys.room.name}!`;
}

donde mys es un objeto queue pasa para obtener los valores